bigtable_automation/gcf/prod/branch.yaml

```diff
@@ -1,8 +1,8 @@
 projectID: "datcom-store"
 instance: "prophet-branch-cache"
 cluster: "prophet-branch-cache-c1"
-nodesHigh: "3"
-nodesLow: "1"
+nodesHigh: "30"
+nodesLow: "3"
```

Comment:
I'm not sure that this need to be 30?  5?

Reply:
Done